lib/Select.js

```diff
@@ -97,7 +98,10 @@ var Select = React.createClass({
 	componentWillMount: function componentWillMount() {
 		this._optionsCache = {};
 		this._optionsFilterString = '';
-		this.setState(this.getStateFromValue(this.props.value));
+		if(canUseDOM) {
+			this.setState(this.getStateFromValue(this.props.value));
```

Comment:
Does this error actually fire here? If so, that is a bug in React 0.13.

The issue in facebook/react#3620 refers to callbacks to OTHER parent components from componentWillMount.

It should be fine to setState in componentWillMount, even on the server.

Perhaps one of the other callback fires or there's a bug in React 0.13 that can be fixed?